README.md

```diff
@@ -2,20 +2,10 @@
 
 Consolidated repo for building and deploying Solo docs. 
 
-## Gloo
-
-The consolidated open source and enterprise Gloo docs are in the `gloo` directory. The content of this directory includes:
-
-* Autogenerated API reference docs, produced during code generation for Gloo based on the protocol buffer definitions. These 
-live in the `docs/v1` directory, and new API docs are pushed here every time Gloo open source or enterprise are released. The 
-files in this directory should not be modified manually. 
-* Autogenerated CLI reference docs, produced for Gloo based on the CLI implementation. These live in the `docs/cli` directory, and
-new CLI docs are pushed here every time Gloo open source or enterprise are released. These files should also not be modified manually. 
-* Manually created docs, static content, and theme overrides. 
-
-## Sqoop
-
-The Sqoop docs are in the `sqoop` directory. 
+# Deprecation notice:
+- Gloo docs are now managed in the [Gloo Repo](https://github.com/solo-io/gloo/tree/feature-rc1)
```

Comment:
this link will be out of date by end of week when we renamed `feature-rc1` branch. can we update this to reflect that?